modules/node_modules/@ncigdc/uikit/Dropdown.js

```diff
@@ -35,7 +37,7 @@ const Dropdown = ({
         onMouseUp={mouseUpHandler}
         onClick={() => setTimeout(() => setActive(false))}
       >
-        {Children.map(children, child =>
+        {Children.map(_.filter(children, React.isValidElement), child =>
```

Comment:
Nope. You can tell by the function signature. Array#map takes a callback as the first argument

https://facebook.github.io/react/docs/react-api.html#react.children